### ISSUE TITLE
Self extractor on ARM seems to pick up the wrong offset.

### ISSUE THREAD

**Issue by fullspectrumdev** [stealth/crash#6]
This is a weird bug, I've no idea why its happening and I've yet to cross build gdb for that target machine.

TL;DR: when ran in self-extract mode, on certain ARMv4l targets, the 'extractor' routine seems to pick up on some part of the binary before the keys begin, and gets to work, producing a 300kb file in /tmp with a load of garbage and then the keys.

I'd hazard a wild guess that somehow, somewhere, there are some strings in the ARM binary that I made which crashd's extractor confuses for the beginning of the stuff to go extract, and just starts doing its job a bit too soon. 

I have no suggestions at this time for how to fix this, I'll upload the build, the build with the keys and the file it chucks out later.

**Comment by stealth** [comment]
You and your static builds. I have to think about a solution.
